### PR TITLE
Easycore-0.3.0

### PR DESCRIPTION
**Pull request by arm61**
This PR gets everything *working* with EasyCore-0.3.0 but does not take advantage of any new features except the MultiFitter.

### CODE REVIEW

**Review by rozyczko — APPROVED** [review]
looks good to me

**Comment by arm61** [comment]
Once https://github.com/easyScience/EasyReflectometryLib/pull/58 is merged I will push the last change on this (moving the EasyReflectometryLib branch to develop) and this is good to merge.

File: EasyReflectometryApp/Gui/Components/ResultsDialog.qml [hunk 1]
```diff
@@ -36,7 +36,7 @@ EaElements.Dialog {
         EaElements.Label {
             enabled: gotResults
             text: gotResults
-                  ? `Goodness-of-fit: ${ExGlobals.Constants.proxy.fitter.fitResults.GOF.toFixed(2)}`
+                  ? `chi<sup>2</sup>: ${ExGlobals.Constants.proxy.fitter.fitResults.chi2.toFixed(2)}`
                   : ""
```
Comment (rozyczko) [inline EasyReflectometryApp/Gui/Components/ResultsDialog.qml:40]: chi2 would look nicer if properly rendered with  `\u03c7\u00b2`

**Review by rozyczko — COMMENTED** [review]
One minor comment for nicer looking display

**Pull request merged**